apps/communications/dialer/js/call_info.js

```diff
@@ -68,34 +77,44 @@
       return;
     }
 
+    if(_showMoreButtonClicked == false) {
+      countToBeDisplayed = 9;
+    } else if (_showMoreButtonClicked == true ) {
+      // Hide the showMoreButton once it is clicked.
+      showMoreButton.classList.add('hide-show-more');
```

Comment:
Those are not list though. They are div's. Doing this thing with div's will be hacky I think ?